uint16 id
float64 hssEnlem
float64 hssBoylam
float64 hssYaricap